### ISSUE TITLE
Missing dependency with 4.6.1 library

### ISSUE THREAD

**Issue by Poimen** [G-Research/consuldotnet#9]
We are attempting to use the .Net Framework 4.6.1 library in a .Net Framework 4.6.2. 

When we install version 0.7.2.6, all works as expected.

When we install version 1.6.1, the following problems occur:
1. Visual Studio does not recognise the package dll properly (there is a yellow triangle on the references tab)

2. Fixing 1 above by manually including the assembly, running a console app gives:
```
System.IO.FileNotFoundException
  HResult=0x80070002
  Message=Could not load file or assembly 'Consul.dll, Version=1.6.1.0, Culture=neutral, PublicKeyToken=20a6ad9a81df1d95' or one of its dependencies. The system cannot find the file specified.
  Source=<Cannot evaluate the exception source>
  StackTrace:
```

So far I cannot tell what is the real problem here - not too clued up on nuget packages and VS is not showing what it doesn't like. I used IL Spy, but can't tell the missing dependency, so I surmise that the 4.6.1 assembly is not built correctly...

Our console application is really simple:
```c#
    class Program
    {
        static void Main(string[] args)
        {
            var consulClient = new ConsulClient(consulConfig =>
            {
                var address = "http://localhost.consul-server";
                consulConfig.Address = new Uri(address);
            });
            var services = consulClient.Agent.Services().Result.Response;
        }
    }
```

Using version 0.7.2.6, the `services` show the services correctly.

**Comment by jgiannuzzi** [comment]
Thanks for reporting this @Poimen, we will try to reproduce it. Could you please let us know which version of Visual Studio you are using?

**Comment by Poimen** [comment]
I am using:
```
Microsoft Visual Studio Professional 2019
Version 16.4.6
```

**Comment by Poimen** [comment]
I should mention that I did:
 - Create console application - v4.6.2
 - Add Package via NuGet

**Comment by jgiannuzzi** [comment]
I was able to reproduce this issue even without Visual Studio, by
* creating a new project with `dotnet new console`
* changing `TargetFramework` to `net461`
* adding the package with `dotnet add package Consul --version 1.6.1`
* changing `Program.cs` to what @Poimen posted [above](https://github.com/G-Research/consuldotnet/issues/9#issue-634433759)

The issue seems to stem from embedding the Newtonsoft.Json assembly within Consul. This was setup to keep the original project behaviour introduced with https://github.com/PlayFab/consuldotnet/pull/52 way back when.
Building without the embedding step seems to fix the issue.

I'm trying to figure out why merging seems to create problems — if I can't, maybe we'll simply drop that step, as the world has moved on and most projects using Consul would probably depend on a recent enough version of Newtonsoft.Json.

**Comment by jgiannuzzi** [comment]
The issue was indeed with incorrect embedding/merging. PR #10 should fix it. Once fully validated, we will release a new build.

**Comment by jgiannuzzi** [comment]
Fixed by PR #10 and released as [v1.6.1.1](https://github.com/G-Research/consuldotnet/releases/tag/v1.6.1.1)

**Comment by Poimen** [comment]
Thanks for the fast support. Tested it this morning, and all works as expected.
